sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+<!-- dev-tool snippets ignore -->
```

Comment:
@mpodwysocki Could you please share the reason for adding this ignore statement here? I noticed that we are not using any `ts snippet:` in this markdown file, and I don’t see a similar statement in other files that don’t contain snippets.